### PR TITLE
doc improvement: zigbee: update VSC screenshots in ZQSG

### PR DESCRIPTION
**Pull request by greg-fer**
Added new screenshots to Zigbee QSG.
Removed unneeded screenshots.
NCSDK-15243.

Signed-off-by: Grzegorz Ferenc <Grzegorz.Ferenc@nordicsemi.no>

### CODE REVIEW

**Review by kittydepa — APPROVED** [review]
LGTM

File: doc/nrf/ug_zigbee_qsg.rst [hunk 1]
```diff
@@ -145,17 +145,14 @@ To program the samples, complete the following steps:
 #. Add the network coordinator application:
 
    a. In the Welcome panel or on the Welcome page, click :guilabel:`Create a new application from sample`.
```
Comment (Avalei) [inline doc/nrf/ug_zigbee_qsg.rst:147]: This is fine for now, but as a head's up we will be changing this to just say "Create a new application" either in the June or July VSC release.

File: doc/nrf/ug_zigbee_qsg.rst [hunk 2]
```diff
@@ -176,9 +173,6 @@ To program the samples, complete the following steps:
       When the process starts, a Zigbee Coordinator panel appears under the Applications panel.
       The generation process takes some time and you can observe its progress in the :guilabel:`nRF Connect` terminal.
```
Comment (Avalei) [inline doc/nrf/ug_zigbee_qsg.rst:174]:
```suggestion
      The generation process takes some time and you can observe its progress in the :guilabel:`nRF Terminal`.
```

**Review by Avalei — CHANGES_REQUESTED** [review]

**Comment by greg-fer** [comment]
@Avalei , please check again.

**Review by Avalei — APPROVED** [review]

**Comment by greg-fer** [comment]
@shantha-14 , @anangl , ready for merge, despite Twister.

**Pull request merged**